e2e/various-suite/frontend-sandbox-datasource.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { random } from 'lodash';
+
+import { e2e } from '../utils';
+
+const DATASOURCE_ID = 'sandbox-test-datasource';
+let DATASOURCE_CONNECTION_ID = '';
+const DATASOURCE_TYPED_NAME = 'SandboxDatasourceInstance';
+
+describe('Datasource sandbox', () => {
+  before(() => {
+    e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'), true);
+
+    e2e.pages.AddDataSource.visit();
+    e2e.pages.AddDataSource.dataSourcePluginsV2('Sandbox datasource test plugin')
+      .scrollIntoView()
+      .should('be.visible') // prevents flakiness
+      .click();
+    e2e.pages.DataSource.name().clear();
+    e2e.pages.DataSource.name().type(DATASOURCE_TYPED_NAME);
+    e2e.pages.DataSource.saveAndTest().click();
```

Comment:
I think the flow `e2e.flows.addDataSource` basically does all of the above.

Reply:
@sunker it does, but in this specific case I need to run validations in the form.